app/assets/stylesheets/concepts.scss

```diff
@@ -165,4 +165,10 @@ div.synonym-change-request button {
 #concepts_json_link{
   margin-top: -4px;
   margin-right: 5px;
+}
+.tree-view-reuse-icon{
```

Comment:
is this specific to ontology concepts? (anyway it should work for all the tree views, concepts, properties, ...)

Reply:
I'll ask Clement if we need to do this for properties also

Reply:
Not only properties but also instances,  schemes, and collections.
and in the concepts it needs to be done, the tree view, the list view, and the date view.

Reply:
@jonquet ?

Reply:
Indeed, the tree view of the Classes/Concept is the one we think by default, but we should think this more gloabally. 

In an ontology, objects reused from another source could be of any type ... especially properties maybe alos instances. 
Scheme and collections the situation would be super rare, but not impossible.